behat_suites.yml

```diff
@@ -7,21 +7,29 @@ repository-forms:
                 - vendor/ezsystems/repository-forms/features/ContentEdit
             contexts:
                 - EzSystems\RepositoryForms\Behat\Context\ContentEditContext
-                - EzSystems\RepositoryForms\Behat\Context\ContentTypeContext
+                - EzSystems\RepositoryForms\Behat\Context\ContentTypeContext:
+                    permissionResolver: '@eZ\Publish\API\Repository\PermissionResolver'
```

Comment:
This wasn't here before, even though Context already had arguments, so I didn't expect it's needed.

Reply:
I was rather thinking about things like https://github.com/ezsystems/ezplatform-admin-ui/blob/master/behat_suites.yml#L36 as when removed dependency by annotations this is no longer built properly.

Reply:
as for one line change, injection with annotations does not work correctly (or I just couldn't find a way with service names using FCQN).

Reply:
@ViniTou it's fixed by https://github.com/ezsystems/ezpublish-kernel/pull/2750

If you can wait a bit so I can merge it then using that would be the preffered approach, as keeping Context definitions in multiple behat_suites file is a tiresome thing.